tools/rosmaster/src/rosmaster/master_api.py

```diff
@@ -73,8 +73,6 @@
 from rosmaster.registrations import RegistrationManager
 from rosmaster.validators import non_empty, non_empty_str, not_none, is_api, is_topic, is_service, valid_type_name, valid_name, empty_or_valid_name, ParameterInvalid
 
-NUM_WORKERS = 3 #number of threads we use to send publisher_update notifications

```

Comment:
This is public API and can't be removed without potentially breaking existing code. Please keep it as-is.